mediator/med_phases_prep_glc_mod.F90

```diff
@@ -20,7 +20,7 @@ module med_phases_prep_glc_mod
   use ESMF                  , only : ESMF_FieldBundleCreate, ESMF_FieldBundleIsCreated
   use ESMF                  , only : ESMF_Field, ESMF_FieldGet, ESMF_FieldCreate
   use ESMF                  , only : ESMF_Mesh, ESMF_MESHLOC_ELEMENT, ESMF_TYPEKIND_R8, ESMF_KIND_R8
-  use ESMF                  , only : ESMF_DYNAMICMASK, ESMF_DynamicMaskSetR8R8R8, ESMF_DYNAMICMASKELEMENTR8R8R8 
+  use ESMF                  , only : ESMF_DYNAMICMASK, ESMF_DynamicMaskSetR8R8R8, ESMF_DYNAMICMASKELEMENTR8R8R8
```

Comment:
are those correct? why do we have multiple R8 at the end?

Reply:
Yes, they seem to be correct (see section [28.6.1](http://earthsystemmodeling.org/docs/release/ESMF_8_0_1/ESMF_refdoc/node5.html#SECTION05056100000000000000))